ietf/templates/ipr/search_doc_result.html

```diff
@@ -54,16 +54,16 @@
             </tr>
         </thead>
 
-        {% for doc in docs %}
+        {% for d in docs %}
             <tbody>
                 <tr>
                     <th scope="col" class="table-info" colspan="3">
-                        Results for {{ doc.name|prettystdname|urlize_ietf_docs }} ("{{ doc.title }}"){% if not forloop.first %}{% if doc.related %}, which was {{ doc.relation|lower }} {{ doc.related.source|prettystdname|urlize_ietf_docs  }} ("{{ doc.related.source.title }}"){% endif %}{% endif %}
+                        Results for {{ d.name|prettystdname|urlize_ietf_docs }} ("{{ d.title }}"){% if d != doc %}{% if d.related %}{% if d != d.related.source %}, which was {{ d.relation|lower }} {{ d.related.source|prettystdname|urlize_ietf_docs }} ("{{ d.related.source.title }}"){% else %}, which {{ d.relation|lower}} {{ d.related.target|prettystdname|urlize_ietf_docs }} ("{{ d.related.target.title }}"){% endif %}{% endif %}{% endif %}
```

Comment:
_shudder_ 

I think maybe the following refactor reduces the 🍝 factor a little?
```
Results for {{ d.name|prettystdname|urlize_ietf_docs }}
("{{ d.title }}"){% if d != doc and d.related %}, which
    {% if d == d.related.source %}
        {{ d.relation|lower }}
        {{ d.related.target|prettystdname|urlize_ietf_docs }}
        ("{{ d.related.target.title }}")
    {% else %}
        was {{ d.relation|lower }}
        {{ d.related.source|prettystdname|urlize_ietf_docs }}
        ("{{ d.related.source.title }}")
    {% endif %}
{% endif %}
```

Reply:
It introduces a lot of whitespace in the resulting html - something Lars was pushing back against pretty hard.

Reply:
The refactor suggested above is in. We can either keep it or revert it.

Reply:
https://github.com/ietf-tools/datatracker/pull/7836/commits/6e4c8038321b6ded5b65ea66ea77143e5d76c06b shows how many whitespace characters were introduced by the previous commit.

Reply:
Seems ok to me. I don't want to pick a fight with Lars though. :-)